tests/trx_finality_status_test.py

```diff
@@ -150,16 +150,21 @@ def isState(state, expectedState, allowedState=None, notAllowedState=None):
     status.append(copy.copy(retStatus))
     startingBlockNum=testNode.getInfo()["head_block_num"]
 
-    def validateTrxState(status, present):
+    def validateTrxState(status):
+        Print(f"status: {status}")
+        present = True
+        state = getState(status)
+        if state == localState or state == unknownState:
+            present = False
         bnPresent = "block_number" in status
         biPresent = "block_id" in status
         btPresent = "block_timestamp" in status
-        desc = "" if present else "not "
+        desc = "" if present else " not"
         group = bnPresent and biPresent and btPresent if present else not bnPresent and not biPresent and not btPresent
         assert group, \
             f"ERROR: getTransactionStatus should{desc} contain \"block_number\", \"block_id\", or \"block_timestamp\" since state was \"{getState(status)}\".\nstatus: {json.dumps(status, indent=1)}"
 
-    validateTrxState(status[0], present=False)
+    validateTrxState(status[0])
```

Comment:
Instead, suggestion would be to calculate `present` before the call to `validateTrxState` in this specific instance where it could have 2 values.  Something like:

`present = True if state == inBlockState else False`
`validateTrxState(status[0], present=present)`